modules/cli/cli.go

```diff
@@ -88,10 +90,12 @@ type CLI struct {
 }
 
 type flagsForFurtherEvaluation struct {
-	DisableLeaderElection    bool
-	ClusterCASecretNamespace string
-	ValidatingWebhookEnabled bool
-	Version                  bool
+	DisableLeaderElection              bool
+	ClusterCASecretNamespace           string
+	ValidatingWebhookEnabled           bool
+	Version                            bool
+	WebhookCertificateConfigBaseImage  string
+	WebhookCertificateConfigShellImage string
```

Comment:
This shouldn't be necessary as previously mentioned. Just add it to the `Config` struct: https://github.com/Kong/gateway-operator/blob/fa31122c75a4ff06710f7f71ce666c771bfc34f4/modules/manager/run.go#L61